recipes/green/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "green" %}
+{% set version = "2.11.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/CleanCut/green/archive/{{version}}.tar.gz
+  sha256: fb78e51a15872c6a4f2952fc6857c91fef390d89b1d725d85b609c119584c137
+
+build:
+  skip: True  # [not py35]
```

Comment:
Why are you only building on 3.5? The readme says "Supports Python 2.7, and 3.4+"....

Reply:
Oh I see, you were trying to skip 3.5 only. That would be `skip: true  # [py35]`.